.github/workflows/testing.yml

```diff
@@ -37,10 +37,14 @@ jobs:
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade wheel
           python -m pip install --upgrade -r doc/requirements.txt
+          python -m pip install --upgrade sbom2doc
       - name: Build docs
         run: |
           cd doc/
           sphinx-build -b html . _build
+      - name: Generate Markdown for SBOM
+        run: |
+          sbom2doc --input-file cve-bin-tool-py${{ matrix.python }}.spdx --format markdown --output-file cve-bin-tool-py${{ matrix.python }}.html
```

Comment:
```suggestion
```
As elsewhere, removing the sbom->doc translation for now until we're ready to use the results in our docs somewhere.  I *think* this will go into the docs build script rather than in testing.yml in the future.